packages/react/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "carbon-components-react",
   "description": "The Carbon Design System is IBM’s open-source design system for products and experiences.",
-  "version": "7.37.0",
```

Comment:
Seems like the version bumps are patches but the title is `rc.0`? Wasn't sure which one it was 👀

Reply:
@joshblack I was a bit confused by the commit instructions on that
![Screen Shot 2021-06-18 at 2 59 17 PM](https://user-images.githubusercontent.com/32556167/122610810-dd5b4880-d045-11eb-8e69-694d482b6d44.png)

Reply:
I suppose it's just a patch. 👀